scripts/examples/webos_example.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+#
+#    Copyright (c) 2021 Project CHIP Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+# Download webOS OSE NDK
+echo "##### Download webOS OSE NDK #####"
+wget http://10.178.94.187/ndk/opt/webos-sdk-x86_64-cortexa7t2hf-neon-vfpv4-toolchain-2.14.1.g.host.manifest
```

Comment:
what is this IP address? 

SDK should be available for everyone, generally in a sdk image.

Also using `build_examples.py` is preffered to shell scripts.

Reply:
@andy31415 
I appreciate your review.
I will check the way to change this as a public url for everyone.
Also, I'll check to see if I can use build_examples.py for webos example build.
Always thank you.

Reply:
@andy31415 
I had changed the private IP address to public url (sourceforge).

I tried to use git-lfs, but failed due to file size limited to 2.2G.
Fortunately, sourceforge doesn't have file size limit.

I still have a problem about time (about 50~70 minutes) spent for downloading 2.2G.
I think I should find a faster hosting ASAP.

Thank you for your review.